packages/react/src/hooks/useController.ts

```diff
@@ -25,6 +26,9 @@ export function useController<OptionsType, OverlayType extends OverlayBase>(
       if (overlayRef.current) {
         return;
       }
+
+      defineCustomElement();
```

Comment:
Does this need to be done for every present, or can it be called when the controller is created instead? (Ditto for useOverlay)

Reply:
My one concern is that this may introduce a sideeffect, but I can definitely check and get back to you.

Reply:
Moving the `defineCustomElement` call to the root of the useOverlay/useController function does not appear to cause a side effect, so I made that change.

No functional difference, but the overhead of calling `defineCustomElement` multiple times has been removed.